api/src/test/java/org/pmiops/workbench/api/CloudTaskInitialCreditsExpiryControllerTest.java

```diff
@@ -705,7 +715,12 @@ public void handleInitialCreditsExpiry_withMissingUsersInRequest_NoNPE() throws
   }
 
   private DbUser createUser(String email) {
-    return userDao.save(new DbUser().setUsername(email));
+    DbUser user = userDao.save(new DbUser().setUsername(email));
+
+    DbVerifiedInstitutionalAffiliation verifiedInstitutionalAffiliation =
+        createDbVerifiedInstitutionalAffiliation(dbInstitution, user);
+    verifiedInstitutionalAffiliationDao.save(verifiedInstitutionalAffiliation);
```

Comment:
An institution affiliation is needed now, because the active initial credits check looks for institutional bypass.